src/celeritas/phys/ParticleData.hh

```diff
@@ -34,6 +34,7 @@ struct ParticleRecord
     units::MevMass mass;  //!< Rest mass [MeV / c^2]
     units::ElementaryCharge charge;  //!< Charge in units of [e]
     real_type decay_constant;  //!< Decay constant [1/s]
+    bool is_antiparticle;  //!< Antiparticle (negative PDG number)
```

Comment:
Probably a basic/dumb physics question: antiprotons and antielectrons obviously annihilate with regular matter. Do unstable antiparticles decay entirely to stable antiparticles? If not then it's not ideal to dump twice the rest mass into local energy deposition in *all* cases...

Reply:
Since only stable particles can be killed we should be safe either way.. but maybe since we really want to check if it's a positron we should just store the positron ID somewhere?

Reply:
Well, I think this is certainly good enough for the time being since the only antiparticle we have is positron 🙃. The code in the loop is clear enough that when we add other antiparticles we'll know to revisit it.

🙃 + 🙂 → 💥💥